src/components/modules/ModuleCheckout.tsx

```diff
@@ -43,29 +43,37 @@ export const ModuleCheckout = (props: ModuleCheckoutProps) => {
 
   const paymentLogoEndMargin: IOSpacingScale = 12;
 
-  return (
-    <PressableModuleBase onPress={props.onPress}>
-      {/*
-        we don't want to let the `space-between`
-        handle spacing for the logo/text section,
-        so we use a row and a marginEnd on the logo
-      */}
-      <View style={styles.rowCenter}>
-        {props.paymentLogo && (
-          <View style={{ marginEnd: paymentLogoEndMargin }}>
-            <LogoPayment name={props.paymentLogo} />
-          </View>
-        )}
-        <View>
-          <H6>{props.title}</H6>
-          <LabelSmall weight="Regular" color={theme["textBody-tertiary"]}>
-            {props.subtitle}
-          </LabelSmall>
+  const ModuleBaseContent = (
+    <View style={styles.rowCenter}>
+      {props.paymentLogo && (
+        <View style={{ marginEnd: paymentLogoEndMargin }}>
+          <LogoPayment name={props.paymentLogo} />
         </View>
+      )}
+      <View>
+        <H6>{props.title}</H6>
+        <LabelSmall weight="Regular" color={theme["textBody-tertiary"]}>
+          {props.subtitle}
+        </LabelSmall>
       </View>
-      <CTA text={props.ctaText} />
-    </PressableModuleBase>
+    </View>
   );
+
+  if (props.ctaText) {
+    return (
+      <PressableModuleBase onPress={props.onPress}>
+        {/*
+          we don't want to let the `space-between`
+          handle spacing for the logo/text section,
+          so we use a row and a marginEnd on the logo
+        */}
```

Comment:
I think this now belongs to the `ModuleBaseContent`, what do you think?

Reply:
Addressed into 56ad7deff683e86c9f63e0470dc2530f9b182b7a